anchor/network/src/network.rs

```diff
@@ -85,6 +91,20 @@ impl Network {
                             AnchorBehaviourEvent::Gossipsub(_ge) => {
                                 // TODO handle gossipsub events
                             },
+                            // Inform the peer manager about discovered peers.
+                            //
+                            // The peer manager will subsequently decide which peers need to be dialed and then dial
+                            // them.
+                            AnchorBehaviourEvent::Discovery(DiscoveredPeers { peers }) => {
+                                //self.peer_manager_mut().peers_discovered(peers);
+                                log::debug!("Discovered peers: {:?}", peers);
+                                for (enr, _) in peers {
+                                    for tcp in enr.multiaddr_tcp() {
+                                        log::debug!("Dialing peer: {:?}", tcp);
+                                        self.swarm.dial(tcp).unwrap();
```

Comment:
I realise this is first draft but can we do something like the below to dodge the unwrap please: 


```suggestion
if let Err(e) = self.swarm.dial(tcp.clone()) {
 log::error!("Error dialing peer {}: {}", tcp,  e);
 }                                       
```